src/styles/blocks/starts.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__star {
+    box-sizing: border-box;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--0 {
+    display: flex;
+
+    & .stars__star:nth-child(-n + 4) {
+      background-image: url('../images/star-active.svg');
+    }
+  }
+}
```

Comment:
Modifier should still work for all number of stars from 0 to 5

Reply:
I've updated selector for modifier.
` @for $i from 0 through 5 {
    &--#{$i} .stars__star:nth-child(-n + #{$i}) {
      background-image: url('../images/star-active.svg');
    }
  }`